ingestion/tests/cli_e2e/test_cli_quicksight.py

```diff
@@ -44,7 +44,7 @@ def expected_not_included_entities(self) -> int:
         return 6
 
     def expected_not_included_sink_entities(self) -> int:
-        return 6
+        return 7
```

Comment:
Wrong expected number of not inclusions before, modified it as per results from quicksight